recipes/xmscore/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+CONDA_BUILD_SYSROOT:
+  - /opt/MacOSX10.9.sdk        # [osx]
```

Comment:
Did you try without this to allow for conda-forge default configuration? We only override it when there is no way around it.